app_dart/bin/server.dart

```diff
@@ -40,6 +40,8 @@ Future<void> main() async {
       '/api/refresh-chromebot-status':
           RefreshChromebotStatus(config, authProvider),
       '/api/refresh-github-commits': RefreshGithubCommits(config, authProvider),
+      //'/api/refresh-github-branches':
```

Comment:
nit: If this isn't needed anymore, we should remove it

Reply:
definitely.